doc/workflows_and_tools/enrollment_tools/yubikey_enrollment_tools.rst

```diff
@@ -0,0 +1,136 @@
+There, select :ref:`import_oath_csv` and the file you just created.
+
+.. _ykpersgui:
+
+.. index:: Yubikey, Yubikey personalization GUI, Yubikey personalization tool
+
+Yubikey Personalization GUI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can also initialize the Yubikey with the official Yubico personalization GUI
+[#ykpers]_ and use the obtained secret to enroll the Yubikey with privacyIDEA.
+For both AES (Yubico OTP) and OATH-HOTP mode, there are two possibilities to initialize
+the Yubikey with privacyIDEA.
+
+Single token enrollment
+=======================
+
+To initialize a single Yubikey in AES mode (Yubico OTP) use the *Quick* button and
+copy the shown secret labeled with "Secret Key (16 bytes Hex)" to the field *OTP Key*
+on the enrollment form in the privacyIDEA WebUI.
+
+.. figure:: images/ykpers-quick-initialize-aes.png
+   :width: 500
+
+   *Initialize a Yubikey in AES mode (Yubikey OTP)*
+
+.. figure:: images/enroll_yubikey.png
+   :width: 500
+
+   *Enroll a Yubikey AES mode token in privacyIDEA*
+
+In the field "Test Yubikey" push the Yubikey button. This will determine the
+length of the *OTP value* and the field *OTP length* is automatically filled.
+
+.. note::
+    The length of the unique passcode for each OTP is 32 characters at the end
+    of the OTP value. The remaining characters at the beginning of the OTP value
+    form the Public ID of the device. They remain constant for each
+    OTP [#ykotp]_.
+
+    privacyIDEA takes care of separating these parts but it needs to know the
+    complete length of the OTP value to work correctly.
+
+The process is similar for the HOTP mode. You have to deselect *OATH Token Identifier*.
+Copy the shown secret to the HOTP :ref:`hotp_token_enrollment` form in privacyIDEA.
+
+.. figure:: images/ykpers-quick-initialize-oath-hotp.png
+   :width: 500
+
+   *To initialize a single Yubikey in HOTP mode, deselect OATH Token Identifier.*
+
+.. note::
+    In the case of HOTP mode privacyIDEA does not know that the token is a Yubikey.
+    To set an identifying token serial, consider to use the advanced mode with an
+    appropriate file output (see below).
+
+Mass enrollment
+===============
+
+To initialize one or more Yubikeys and to write the configuration to an output file use the
+*Advanced* button. The configured secrets are imported afterwards in the WebUI. After plugging
+the Yubikey, it will be detected and the selected configuration slot is initialized upon
+*Write Configuration* as before. You will be prompted to select an output file name.
+The secrets are written to this file in the format set in *Settings* -> *Log configuration output*.
+We recommend to select *Yubico format* since here privacyIDEA is able to detect the Yubikey mode and
+sets the serial accordingly. PSKC format is also supported upon import. You may also use a custom
+output configuration to set custom token serials upon import using :ref:`import_oath_csv`.
```

Comment:
Please check, if the rest of the seed-file description fits my above comment.

Reply:
done